### DISCUSSION TITLE
IntelliJ Sharing

### DISCUSSION THREAD

**Discussion by Ailachayra** [community/community#130358] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Other

### Body

![gambar](https://github.com/community/community/assets/153358299/84298468-782c-4501-b85a-04bc4febfc54)
Why i can't share my project on github from intelliJ?😓

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Ailachayra ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
